src/components/atoms/Rail/index.jsx

```diff
@@ -1,28 +1,41 @@
-import { useRef, useEffect } from 'react'
+import { useRef, useEffect } from 'react';
 import styles from './rail.module.css';
+import PropTypes from 'prop-types';
 
 const Rail = (props) => {
   const railRef = useRef(null);
-  const scrollSpeed = typeof props.scrollSpeed === 'number' ? props.scrollSpeed : 1;
+  const scrollSpeed =
+    typeof props.scrollSpeed === 'number' ? props.scrollSpeed : 1;
 
   useEffect(() => {
     const el = railRef.current;
-    const onWheel = e => {
+    const onWheel = (e) => {
       if (e.deltaY == 0) return;
       e.preventDefault();
       el.scrollTo({
-        left: el.scrollLeft + (e.deltaY * scrollSpeed),
-        behavior: "smooth" // "smooth" or "auto"
+        left: el.scrollLeft + e.deltaY * scrollSpeed,
+        behavior: 'smooth', // "smooth" or "auto"
       });
     };
     if (el) {
-      el.addEventListener("wheel", onWheel);
+      el.addEventListener('wheel', onWheel);
     }
-    return () => el && el.removeEventListener("wheel", onWheel);
+    return () => el && el.removeEventListener('wheel', onWheel);
   }, []);
 
-  return (<main className={[props.className, styles.railContainer].join(' ')} ref={railRef}>
-    {props.children}
-  </main>);
-}
-export default Rail;
\ No newline at end of file
+  return (
+    <main
+      className={[props.className, styles.railContainer].join(' ')}
+      ref={railRef}>
+      {props.children}
+    </main>
+  );
+};
+
+Rail.propTypes = {
+  scrollSpeed: PropTypes.number,
```

Comment:
Scroll speed is optional
Don't give proptype as strictly required prop

Reply:
yes i havent made it .isRequired